glide.lock

```diff
@@ -11,6 +11,8 @@ imports:
   version: 346938d642f2ec3594ed81d874461961cd0faa76
   subpackages:
   - spew
+- name: github.com/dgryski/go-farm
+  version: 83948bc0eb076b6b72c28abe5282fa8cf5240db6
```

Comment:
it depends on the io.Writer, since you'd be passing a buffer it won't ever error out